core/client/pom.xml

```diff
@@ -170,46 +165,16 @@
           <groupId>org.apache.curator</groupId>
           <artifactId>curator-client</artifactId>
```

Comment:
could you update the comment for spark profile
- indicate curator-client, curator-framework, curator-recipies and hadoop-client are marked as provided as they use different versions of guava and jboss netty.
- jackson-core and jackson-databind are also marked as provided, do we know the reason? if yes, better put here